src/utils.rs

```diff
@@ -1,30 +1,14 @@
-use anyhow::anyhow;
 use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
 use fvm_ipld_encoding::RawBytes;
 use fvm_shared::address::Address;
-use fvm_shared::bigint::bigint_ser;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::MethodNum;
-use serde::de;
-
-/// A macro to abort concisely.
-/// This should be part of the SDK as it's very handy.
-#[macro_export]
-macro_rules! abort {
-    ($code:ident, $msg:literal $(, $ex:expr)*) => {
-        fvm_sdk::vm::abort(
-            fvm_shared::error::ExitCode::$code.value(),
-            Some(format!($msg, $($ex,)*).as_str()),
-        )
-    };
-}
 
 #[derive(Serialize_tuple, Deserialize_tuple, Clone, Debug)]
```

Comment:
This can go, it was used for testing purposes, and we already have a `expectSend`as part of the `runtime`